client/src/index.css

```diff
@@ -0,0 +1,14 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
Good idea for making text lighter on dark mode backgrounds. Also nice to see [cross browser compatibility](https://en.wikipedia.org/wiki/Computer_programming#Readability_of_source_code) taken into account.